docs/operations/data_migration.md

```diff
@@ -78,55 +78,241 @@ An example script using this technique is in
+
+NOTE: If you'd wish to change what directory these files are uploaded to, you can change `ssh.fr.cloud.gov:tmp/` to `ssh.fr.cloud.gov:{DIRECTORY_YOU_WANT}/`, but be aware that this makes data migration more tricky than it has to be.
+
+##### Get a temp auth code
+
+The scp command requires a temporary authentication code. Open a new terminal instance (while keeping the current one open),
+and enter the following command:
+```shell
+cf ssh-code
+```
+Copy this code into the password prompt from earlier.
+
+NOTE: You can use different utilities to copy this onto the clipboard for you. If you are on Windows, try the command `cf ssh-code | clip`. On Mac, this will be `cf ssh-code | pbcopy`
+
+#### STEP 2: Transfer uploaded files to the getgov directory
+Due to the nature of how Cloud.gov operates, the getgov directory is dynamically generated whenever the app is built under the tmp/ folder. We can directly upload files to the tmp/ folder but cannot target the generated getgov folder directly, as we need to spin up a shell to access this. From here, we can move those uploaded files into the getgov directory using the `cat` command. Note that you will have to repeat this for each file you want to move, so it is better to use a tar.gz for multiple, and unpack it inside of the `datamigration` folder.
+
+##### SSH into your sandbox
+
+```shell
+cf ssh {FULL_NAME_OF_YOUR_SANDBOX_HERE}
+```
+
+##### Open a shell
+
+```shell
+/tmp/lifecycle/shell
+```
+
+##### From this directory, run the following command:
+```shell
+./manage.py cat_files_into_getgov --file_extension txt
+```
+
+NOTE: This will look for all files in /tmp with the .txt extension, but this can
+be changed if you are dealing with different extensions. For instance, a .tar.gz could be expressed
+as `--file_extension tar.gz`.
+
+If you are using a tar.gz file, you will need to perform one additional step to extract it.
+Run the following command from the same directory:
+```shell
+tar -xvf migrationdata/{FILE_NAME}.tar.gz -C migrationdata/ --strip-components=1
+```
+
+*FILE_NAME* - Name of the desired file, ex: exportdata
+
+
+
+#### Manual method
+If the `cat_files_into_getgov.py` script isn't working, follow these steps instead.
+
+##### Move the desired file into the correct directory
+
+```shell
+cat ../tmp/{filename} > migrationdata/{filename}
+```
+
+
+*You are now ready to run migration scripts (see "Running the Migration Scripts")*
+
+### SECTION 2 - LOCAL MIGRATION SETUP (TESTING PURPOSES ONLY)
+
+***IMPORTANT: only use test data, to avoid publicizing PII in our public repo.***
+
+In order to run the scripts locally, we need to add the files to a folder under `src/`.
 This will allow Docker to mount the files to a container (under `/app`) for our use.  
 
  - Create a folder called `tmp` underneath `src/`
  - Add the above files to this folder
  - Open a terminal and navigate to `src/`
```

Comment:
This may no longer be valid, since we have the "migrationdata" folder now. Marking as a note for myself to update this

Reply:
Did you two branch off of this PR? if so it should make it find to pull in